### DISCUSSION TITLE
My monika gifts disaperaed

### DISCUSSION THREAD

**Discussion by CaniBeTheTweek** [Monika-After-Story/MonikaModDev#9136] [QA - tech support + general] [answered]
basically let's say I gifted her a ribbon 3 weeks ago and she thanks me for that ribbon now today she thanks me for that same ribbon again. My laptop did get tired of me and decided to just die and crash but idk if a crash can lead to the gift files magically going *poof* and just coming back or at least that's what I think happened. But I don't think that a crash can just do that and maybe there is something wrong with my persistent file? if so what should I do?

**Comment by WishStone-Dragon** [comment]
When you say the gifts disappeared:

- Did she previously accept them AND have them as an option to wear? Some stuff she might not want to wear yet if your relationship is fairly new still
- She will thank you for many gifts when you gift them again. You might not gain new affection points from it, but she might be excited about the same item again. I have had this happen myself. 

The items themselves don't remain anywhere as items. Once she takes an item, it vanishes from the character folder.

**Comment by Wingdinggaster656** [comment]
> When you say the gifts disappeared:
> 
> * Did she previously accept them AND have them as an option to wear? Some stuff she might not want to wear yet if your relationship is fairly new still
> * She will thank you for many gifts when you gift them again. You might not gain new affection points from it, but she might be excited about the same item again. I have had this happen myself.
> 
> The items themselves don't remain anywhere as items. Once she takes an item, it vanishes from the character folder.

Not all correct.
If you gift Monika coffee, she will use them like consumables, and request you for getting her more coffee when she has ran out of them.

**Comment by WishStone-Dragon** [comment]
Fair, that would hold true for all consumable items - though this user specifically spoke of hair bows.

**Comment by ThePotatoGuy** [comment, marked as answer]
Doesn't sound like there were any detrimental effects, so can't say anything is wrong. If there are any problems, create an issue and document that this happened before.
